external/sketch/ansible/roles/ingress-restrictions/README.md

```diff
@@ -0,0 +1,24 @@
+ingress-restrictions
+=========
+
+This role is used to help reduce the chances for backflip correlation from public ssh keys from tools like Censys.
+
+This will automatically tell middle hosts to only allow connections on port 2222 from edge hosts.
+
+This role only handles the sketch infrastructure as of now.
+
+Requirements
+------------
+
+Once you've ran this in your Sketch infrastructure, you will need to grab the IP Addresses or IP Address range from your middles and add them to your OCI/AWS infrastructure manually.
```

Comment:
How do you do this? This seems to be a critical part of fixing #70

Reply:
It may not fully resolve #70 given we had to setup 1:1:1 correlation of proxy, middle, edge in an engagement. However, outside of that, we are blocking 2222 on all of Sketch using this. The only manual part is taking our middles and throwing them into OCI/AWS to allow there from middles and block elsewhere.

Reply:
If it doesn't fully resolve #70, is there a new ticket for the remainder of the work? Without it fully resolved, there is still an opsec leak unless that manual process is performed - which should be clearly documented for now.

Reply:
https://github.com/redteaminfra/redteam-infra/issues/70#issuecomment-2101016861

Reply:
I've broken up the issue into two parts and renamed this one. #70 is now only for the work handled by this role while #114 covers the remainder work and enchancements.